test/JDBC/expected/babel-4475-vu-prepare.out

```diff
@@ -0,0 +1,11 @@
+create view babel_4475_v1 as select cast(cast('a' as binary(2)) as varchar(2))
```

Comment:
we should call the cast from view, function and procedures. Also, we have 4 casts, there should be corresponding 4 tests. We should add a test which validates typmod and maxlen with cases like varchar, varchar(max), varchar(1), varchar(x) and value larger than limit

Reply:
Added.